mlflow/ml-package-versions.yml

```diff
@@ -568,6 +568,7 @@ transformers:
       # See: https://github.com/huggingface/transformers/issues/23352 for discussion
       "< 4.30.0": ["tensorflow<2.13.0"]
     pre_test: |
+      export PYTHONPATH=./
```

Comment:
This is necessary for python to discovery tests/ module path, simply adding `__init__.py` file doesn't work